# x position error (float)
float32 x_error

# y position error (float)
float32 y_error

# theta position error (float)
float32 theta_error